src/core_modules/capture-core/components/WidgetFeedback/WidgetFeedback.component.js

```diff
@@ -1,23 +1,29 @@
 // @flow
-import React, { useCallback, useState } from 'react';
+import React, { useState } from 'react';
 import i18n from '@dhis2/d2-i18n';
 import { Widget } from '../Widget';
 import type { Props } from './WidgetFeedback.types';
+import { WidgetFeedbackContent } from './WidgetFeedbackContent/WidgetFeedbackContent';
+
+export const WidgetFeedback = ({ feedback, emptyText }: Props) => {
+    const [openStatus, setOpenStatus] = useState(true);
 
-export const WidgetFeedback = ({ className }: Props) => {
-    const [openStatus, setOpenStatus] = useState(false);
     return (
         <div
-            className={className}
+            data-test={'feedback-widget'}
```

Comment:
Hey, there is no need to wrap the data test id inside the `{ }`. Thanks!

Reply:
Thanks, updated it in the [Indicator-PR](https://github.com/dhis2/capture-app/pull/1831) 👍